src/Spec/Swagger2.php

```diff
@@ -337,6 +338,18 @@ public function getDefinitions()
                     }
                 }
             }
+            if (isset($sch['error_types'])) {
+                $types = [];
+                foreach ($sch['error_types'] as $type) {
+
+                    $types[] = [
+                        'code' => $type['code'],
+                        'type' => $type['type'],
+                        'description' => $type['description']
```

Comment:
in the spec, the key for this is actually `message`. We'll need to finalize what it's going to be in the spec and make sure it matches over here.

Reply:
Ok for now i'll change it to message
`'message' => $type['message']`